tests/mock_network/churn.rs

```diff
@@ -20,18 +20,11 @@ use routing::{
 };
 use std::collections::{BTreeMap, BTreeSet, HashMap, HashSet};
 
-/// Randomly removes some nodes, but <1/3 from each section and never node 0.
-/// Never trigger merge: never remove enough nodes to drop to `elder_size`.
-/// max_per_pfx: limits dropping to the specified count per pfx. It would also
-/// skip prefixes randomly allowing sections to split if this is executed in the same
-/// iteration as `add_nodes_and_poll`.
+/// Randomly removes some nodes, but <1/3 from each section. Never remove enough nodes to drop to
```

Comment:
Is that 1/3 ok? I think we drop all in one go, 1/3 or all nodes in a section may be including all elders?

Reply:
I think the function accounts for it. Maybe the comment needs to be more precise?

Reply:
I cannot really see it, but if you can tell me where and update the comment that would do.